download/src/androidTest/java/com/cloudinary/android/download/DownloadRequestBuilderImplTest.java

```diff
@@ -0,0 +1,119 @@
+package com.cloudinary.android.download;
+
+import android.content.Context;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+
+import com.cloudinary.Transformation;
+import com.cloudinary.android.MediaManager;
+import com.cloudinary.android.ResponsiveUrl;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+
+import com.cloudinary.android.download.test.R;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DownloadRequestBuilderImplTest {
+
+    private static final String TEST_CLOUD_NAME = "demo";
+    private static final String TEST_PUBLIC_ID = "sample";
+
+    private DownloadRequestBuilderImpl sut;
+
+    @Mock
+    private DownloadRequestBuilderStrategy downloadRequestBuilderStrategy;
+
+    @BeforeClass
+    public static void setup() {
+        Context context = InstrumentationRegistry.getInstrumentation().getTargetContext();
+        MediaManager.init(context);
+        MediaManager.get().getCloudinary().config.cloudName = TEST_CLOUD_NAME;
```

Comment:
The cloud info should be injected from the environment instead of hard coded constants.
You can see the full workflow in the core tests. The test manifest has a template place holder and the build process replaces it with with the env variable.

https://github.com/cloudinary/cloudinary_android/blob/master/core/src/androidTest/AndroidManifest.xml and 
https://github.com/cloudinary/cloudinary_android/blob/master/core/build.gradle (the `manifestPlaceholders ` line).

Let me know if you need assistance with that.